### PR TITLE
fix current test issues so we can merge

### PR DESCRIPTION
**Pull request by david-caro**
- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [x] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [x] I've added any new docs if API/utils methods were added.
- [x] I have updated the existing documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

As it includes the python requests version, better ignore it.

Signed-off-by: David Caro <david@dcaro.es>

### CODE REVIEW

**Comment by david-caro** [comment]
This works locally, but it seems to fail in travis :/

**Comment by david-caro** [comment]
ci merge please

**Comment by david-caro** [comment]
ci merge please

**Comment by david-caro** [comment]
ci merge please

**Comment by david-caro** [comment]
Dammit.... looking

**Comment by david-caro** [comment]
ci merge please

File: inspirehep/testlib/api/base_resource.py [hunk 1]
```diff
@@ -34,3 +34,12 @@ def __repr__(self):
...
         return '{classname}({args})'.format(classname=classname, args=args)
+
+    def __eq__(self, other):
+        for this_prop, other_prop in zip(dir(self), dir(other)):
+            if this_prop != other_prop:
+                return False
+
+        return len(dir(self)) == len(dir(other))
+
+        return True
```
Comment (michamos) [inline inspirehep/testlib/api/base_resource.py]: you never get to this line, as you have an unconditional `return` just above.

Comment (david-caro) [reply inspirehep/testlib/api/base_resource.py]: Too late :/, opened another pr for it

**Pull request merged**